docs/canonicalk8s/charm/howto/ceph-csi.md

```diff
@@ -180,8 +194,8 @@ associated namespace. Ensure the configuration for the `ceph-csi-alt`
 application changes so that it doesn't collide with `ceph-csi`.
```

Comment:
```suggestion
application is set so that it doesn't collide with `ceph-csi`. In this example we have deployed the `ceph-csi` in `ceph-ns` namespace and `ceph-csi-alt` in `ceph-csi-alt` namespace.
```
then check it is in a different ns rather than setting the ns as we should already have it set from deploying it